S/SQLite/build_tarballs.jl

```diff
@@ -3,14 +3,15 @@
 using BinaryBuilder, Pkg
 
 name = "SQLite"
-version = v"3.39.2"
+version = v"3.39.4"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://www.sqlite.org/2022/sqlite-autoconf-3390200.tar.gz",
-                  "852be8a6183a17ba47cee0bbff7400b7aa5affd283bf3beefc34fcd088a239de"),
-    FileSource("https://git.archlinux.org/svntogit/packages.git/plain/trunk/license.txt?h=packages/sqlite&id=33cad63ddb1ba86b7c5a47430c98083ce2b4d86b",
-               "4e57d9ac979f1c9872e69799c2597eeef4c6ce7224f3ede0bf9dc8d217b1e65d"; filename="LICENSE"),
+    ArchiveSource("https://www.sqlite.org/2022/sqlite-autoconf-3390400.tar.gz",
+                  "f31d445b48e67e284cf206717cc170ab63cbe4fd7f79a82793b772285e78fdbb"),
+    FileSource("https://github.com/archlinux/svntogit-community/blob/cf0a3337bd854104252dc1ff711e95cc8bc7ffb3/trunk/license.txt",
```

Comment:
This is an html page, isn't it? You want the raw file

Reply:
Corrected.